documentation/modules/exploit/multi/browser/msfd_rce_browser.md

```diff
@@ -0,0 +1,103 @@
+## Description
+
+This module connects to the Metasploit msf daemon and uses the ruby interpreter
+to achieve a shell. This module can be used when the vulnerable service is only
+listening on localhost and the attacker has not achieved a foothold on the
+machine. To achieve code execution, the data is sent from the victim's browser.
+To achieve this, the victim must visit the attacker's web site.
+
+## Vulnerable Application
+
+[Metasploit](https://github.com/rapid7/metasploit-framework) has a legacy method
+of providing remote access. The msf daemon provides no means of authentication
+and is therefore vulnerable against an attacker who has IP access to the daemon.
+In those cases where the daemon is listening on localhost, we take advantage of
+the fact that the victim's web browser has IP access to the daemon.
+
+Because the msfconsole-program is line-based, it is possible to successfully
+send commands through the HTTP-protocol. This module takes advantage of this by
+placing the msfconsole-command in the POST-data of a HTTP-request. Since these
+POST-requests are considered safe to send cross-domain, it is possible to send
+them from the attacker's web application and to localhost using the victim's
+browser.
+
+To start the vulnerable service, run:
+
+```
+$ msfd -f -q
+```
+
+This module has been tested on the following targets:
+
+* Metasploit 5.0.0 on Ubuntu 16.04 (using Firefox and Chrome)
+* Metasploit 4.16.51 on Windows 7 (using IE, exploitation failed in Firefox and
```

Comment:
If it affects older versions of metasploit, write `< 4.16.51`

Reply:
I've tried to clarify on this point. Pretty much all versions are vulnerable, but I have only tested the targets listed.